Sources/TuistGenerator/Utils/SwiftPackageManagerInteractor.swift

```diff
@@ -54,7 +54,7 @@ public class SwiftPackageManagerInteractor: SwiftPackageManagerInteracting {
         graphTraverser: GraphTraversing
     ) async throws {
         guard !config.generationOptions.disablePackageVersionLocking,
-              graphTraverser.hasRemotePackages
+              graphTraverser.hasPackages
```

Comment:
Can we also add a unit test in `SwiftPackageManagerInteractorTests` to test this behavior?

Reply:
Done!